doc/source/index.rst

```diff
@@ -23,9 +23,11 @@ Contributing
 
 * `Source code`_
 * `Issue tracker`_
+* Join us on `Discord`_
 
 .. _`Source code`: https://github.com/PyCQA/bandit
 .. _`Issue tracker`: https://github.com/PyCQA/bandit/issues
+.. _`Discord`: https://discord.gg/qYxpadCgkx
```

Comment:
Believe it or not, these don't need the backticks around them, they can be more simply:

```rest
.. _Source code: ...
.. _Issue tracker: ...
.. _Discord: ...
```

but that's not important

Reply:
I notice in my IDE at least, it will de-highlight the line as if it's a comment or hidden text when using the back ticks. That might help to inform people these lines are not intended to be visible, just for link mapping.

Reply:
I think that's a bug with your IDE not supporting a superior documentation format then ;)